packages/x-date-pickers/src/DateCalendar/DayCalendar.tsx

```diff
@@ -54,7 +54,7 @@ export interface ExportedDayCalendarProps<TDate> extends ExportedPickersDayProps
    */
   loading?: boolean;
   /**
-   * Component displaying when passed `loading` true.
+   * Component rendered on the "day" view when `props.loading` is true.
```

Comment:
Nitpick: Is `props.` really needed here?
Have we used this approach anywhere else? 🤔 
```suggestion
   * Component rendered on the "day" view when `loading` is true.
```

Reply:
I used it in 2022 in the CHANGELOG for `props.data` :laughing: 
We also have it on `usePickerValue` for `props.value`.

My reasoning is that for most props, the name is specific enough to make clear that it's a prop.
But for super generic concepts like `loading`, `value`, `data`, I think explicitly stating that it's a prop and not some internal variable gives additional clarity to the sentence.

Reply:
Thanks for clarifying. 👌 
Agreed, it shouldn't hurt. 👍